http4k-multipart/src/main/kotlin/org/http4k/lens/parts.kt

```diff
@@ -20,8 +20,23 @@ data class MultipartFormField(val value: String, val headers: Headers = emptyLis
     }
 }
 
-data class MultipartFormFile(val filename: String, val contentType: ContentType, val content: InputStream) : Closeable {
-    override fun close() { content.close() }
+data class MultipartFormFile internal constructor(
+    val filename: String,
+    val contentType: ContentType,
+    val content: InputStream,
+    val closeable: Closeable
```

Comment:
Adding the `Closeable` to the data class properties makes it so that the data class `equals` and `hashCode` will no longer work.  Evidently, there are no tests failing due to this--and they would have failed anyway because of the `InputStream`--but it makes me question why this was a `data class` to being with.